### ISSUE TITLE
[Feature Request] Email contents in notification

### ISSUE THREAD

**Issue by darigovresearch** [ProtonMail/proton-mail-android#93]
**Is your feature request related to a problem? Please describe.**
Would be good to see a small preview in the email notification to get context of the email

**Describe the solution you'd like**
Just the first x words of the email to be visible in the android notifications

**Describe alternatives you've considered**
Shouldn't need more than the above

**Additional context**
Most other email clients & android apps have a similar feature, potentially could be an easy community pull request submission

**Comment by gahfy** [comment]
Due to issue #96, notifications are now relying on Firebase (Google Cloud Messaging Service)

To my point of view (and for many others I guess) sender and title looks like already too much. I wouldn't feel safe if content (even partial) of my emails would be sent that way

**Comment by bartbutler** [comment]
@gahfy Keep in mind that push notifications are E2E encrypted to your device, so the notification provider, whether APNS or FCM does not see any data, just an encrypted blob.

**Comment by gahfy** [comment]
@bartbutler I just checked the code of `app/src/main/java/ch/protonmail/android/fcm/FcmIntentService.java`
Sorry for talking too fast, my bad, it is e2e encrypted

**Comment by darigovresearch** [comment]
@gahfy potentially worth having a setting somewhere in the app for the user to decide what they wish to see in the notification. Or have seen where the user by default sees the minimum data but can drag out with two fingers to see more details of the email within the notification.

Would you be able to have a go at making a pull request as you have done for https://github.com/ProtonMail/proton-mail-android/issues/32 ?

**Comment by gahfy** [comment]
Actually this could be done, and I would love to work more on the project. But I'm just waiting for a feedback on my PRs to be sure that time invested on it worth it.

**Comment by darigovresearch** [comment]
Yes totally understand, we tend to also only make one pull request to see if it is merged by the maintainers before putting in more time and effort on additional pull requests
